### DISCUSSION TITLE
Is there an argument to make the container restart regardless of whether it is updated

### DISCUSSION THREAD

**Discussion by jamesriley101** [containrrr/watchtower#1284] [Q&A] [answered]
I want my container to be updated every night if an update is available, but to always restart at that scheduled time whether or not an update is available.

Is this supported, and if so, what argument should be used? Thanks!

**Comment by zoispag** [comment, marked as answer]
No, this is not possible. Watchtower only acts on image updates. You can do two things (that I can think off):
1. Add a cron task on your host
2. Create a container with cron (look for supercronic) which will have access to docker, by mounting the docker.sock.

Both will result in manual update on predefined time. 1st one is simpler, but requires to perform the same action on all hosts. Second one is more difficult to implement, but easier to maintain.

If you really need to act only in case watchtower did not restart the container, you can also inspect the running container to know when it was started.

**Comment by Kilobites-cmd** [comment]
please fill all the important forms of my documents
